### PR TITLE
feat(HSS): support new resource to manage CCE protection of HSS

### PR DESCRIPTION
**Pull request by deer-hang**
**What this PR does / why we need it**:

- Support new resource to manage CCE protection of HSS.
  - Due to the limitations of the test environment, only some scenarios and abnormal situations can be tested.
  - The current test cases cannot be executed 100% successfully. We need to re-verify after the open API is launched on the official website.
  - Currently, the test case can only be called in the `cn-north-7` region.
  - Because the API has not been released to the official environment, no documentation is provided.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.

```
make testacc TEST='./huaweicloud/services/acceptance/hss' TESTARGS='-run TestAccCCEProtection_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/hss -v -run TestAccCCEProtection_basic -timeout 360m -parallel 4
=== RUN   TestAccCCEProtection_basic
=== PAUSE TestAccCCEProtection_basic
=== CONT  TestAccCCEProtection_basic
--- PASS: TestAccCCEProtection_basic (169.03s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/hss       170.141s
```

* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

File: huaweicloud/services/hss/resource_huaweicloud_hss_cce_protection.go [hunk 1]
```diff
@@ -0,0 +1,569 @@
...
+	)
+
+	client, err := cfg.NewServiceClient(product, region)
+	if err != nil {
+		return diag.Errorf("error creating HSS client: %s", err)
+	}
+
+	respBody, err := ReadCCEProtection(client, clusterID, clusterName, epsID, region)
+	if err != nil {
+		return diag.Errorf("error retrieving CCE protection configuration: %s", err)
```
Comment (jinyangyang222) [inline huaweicloud/services/hss/resource_huaweicloud_hss_cce_protection.go:453]:
```suggestion
		return diag.Errorf("error retrieving HSS protection configuration for CCE: %s", err)
```

Comment (jinyangyang222) [reply huaweicloud/services/hss/resource_huaweicloud_hss_cce_protection.go:453]: There are similar issues in other places as well

Comment (deer-hang) [reply huaweicloud/services/hss/resource_huaweicloud_hss_cce_protection.go]: OK

**Pull request closed without merging**